app/ide-desktop/lib/dashboard/src/services/LocalBackend.ts

```diff
@@ -44,29 +175,73 @@ export default class LocalBackend extends Backend {
     }
   }
 
+  /** Return the ID of the root directory. */
+  override rootDirectoryId(): backend.DirectoryId | null {
+    return newDirectoryId(this.defaultRootDirectory)
+  }
```

Comment:
Are we sure we want to allow this to return `null`? How can that happen when we're dealing with the local backend?

Reply:
the types were copy-pasted by VSCode from `Backend`. it's true that this shouldn't ever return `null`.